.github/workflows/auto_cherry_pick.yml

```
@@ -43,3 +43,37 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.to_branch }}
+      - name: Set issue title
+        if: ${{ failure() }}
+        run: |
+            echo "title=[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}" >> "$GITHUB_ENV"
+      - name: Create Issue on Failed workflow
+        if: ${{ failure() }}
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ github.token }}
+          title: |
+            ${{ env.title }}
+          body: |
+            #### Auto-Cherry-Pick WorkFlow Failure:
+
+            - [Failed Run](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+
+            - [PR Number](https://github.com/${{ github.repository }}/pull/${{ github.event.number }})
+
+            - Workflow Name - `${{ github.workflow }}`
+
+            - Job -           `${{ github.job }}`
+
+            - Status -        `${{ job.status }}`
+
+          labels: Failed_AutoCherryPick
+          assignees: "${{ github.event.pull_request.user.login }}"
+
+      - name: Google Chat Notification - Robottelo
+        if: ${{ failure() }}
+        uses: Co-qn/google-chat-notification@releases/v1
+        with:
+          name: ${{ env.title }}
+          url: ${{ secrets.GOOGLE_CHAT_WEBHOOK }}
```

```suggestion
          url: ${{ secrets.GCHAT_REVIEWERS_WEBHOOK }}
```
I would rename this to `GCHAT_REVIEWERS_WEBHOOK` to be specific.

---

@jyejare please add the secret with that name to point to gchat channel webhook

---

@jyejare I added that as an organization secret ! Feel free to use it !